charter/index.md

```diff
@@ -0,0 +1,228 @@
+
+## Scope of work
+
+Under the present charter, the Community Group will
+determine a structure and process
+under which such specifications may be created
+by which the greatest number of interested parties may be able to participate.
+Until such a structure and process is determined,
+no technical development of specifications or implementations
+will be undertaken by this Community Group
+and no technical contributions will be accepted.
+
+## Deliverables
+
+No technical specifications will be produced by this Community Group
+and no technical contributions are
+to be solicited
+or accepted.
+
+* This Community Group aims to produce a proposed Charter,
+to be used to set up a technical group,
+possibly a W3C Working Group,
+under which members of this community should conduct technical work.
+
+* If, after producing such a charter,
+the Community Group determines that its current composition is adequate
+for conducting its work,
+then the charter produced is
+to be adopted as an updated Charter for this CG.
+
+* If the Community Group determines that it would be appropriate
+to divide the technical work into several venues,
+it may produce several such proposed charters
+rather than a single one.
+
+* Various supporting documents and articles
+describing the problem space
+may also be produced.
+
+All deliverables of this Community Group will use the
+[W3C Software and Document License](http://www.w3.org/Consortium/Legal/2015/copyright-software-and-document).
+
+This group maintains an [index](https://w3c.github.io/font-text-cg/work-items/)
+of the documents it is working on.
+
+## Community and Business Group Process and Patent Policy
+
+The group operates under the [Community and Business Group Process](https://www.w3.org/community/about/agreements/).
+Terms of in this charter that conflict with those of the Community and Business Group Process are void.
+
+As with other Community Groups,
+W3C seeks organizational licensing commitments
+under the [W3C Community Contributor License Agreement (CLA)](http://www.w3.org/community/about/agreements/cla/).
+(Technical Proposals in this Community Group charter are applicable “Specification” in the CLA).
+When people request to participate
+without representing their organization’s legal interests,
+W3C will in general approve those requests for this group
+with the following understanding:
+W3C will seek and expect an organizational commitment
+under the CLA starting with the individual’s first request
+to make a contribution to a group deliverable.
+The section on Contribution Mechanics describes
+how W3C expects to monitor these contribution requests.
+
+## Communication
+
+This group operates under the
+[W3C Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/).
+
+This group will conduct its work publicly,
+primarily using the public-font-text@w3.org mailing list
+([archives](https://lists.w3.org/Archives/Public/public-font-text/))
+for minutes and announcements,
```

Comment:
We have minutes posted in this repository as well. Do they need to also be copied to the WC3 mailing list as a matter of procedure? Or would it be okay to continue with our current system of a live-edited Google doc for sketching them out in real time and then archiving them in this repository?

Reply:
They do need to be on the W3C list for archival at W3C, as I understand it.

Reply:
Roger that, it makes sense actually. Carry on.